scripts/run_backend_tests.py

```diff
@@ -358,7 +356,7 @@ def main(args=None):
             all_test_targets = [parsed_args.test_target + '_test']
     elif parsed_args.test_shard:
         validation_error = _check_shards_match_tests(
-            include_load_tests=False)
+            include_load_tests=True)
```

Comment:
If we are going to run a load test all the time, do we need such a flag in the script?  I think it should be fine to remove the  `exclude_load_tests` flag completely? (not sure!)

Reply:
Removing the `exclude_load_tests` flag seems unrelated to this PR, so I think it should be handled in a separate PR

Reply:
Yep, I agree! I just wanted to confirm whether it can be removed now!